ravenframework/Models/PostProcessors/Validations/Representativity.py

```diff
@@ -0,0 +1,434 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+  Created on April 29, 2021
+
+  @author: Mohammad Abdo (@Jimmy-INL)
+
+  This class represents a base class for the validation algorithms
+  It inherits from the PostProcessor directly
+  ##TODO: Recast it once the new PostProcesso API gets in place
+"""
+
+#External Modules------------------------------------------------------------------------------------
+import numpy as np
+import xarray as xr
+import scipy as sp
+from scipy.linalg import sqrtm
+#External Modules End--------------------------------------------------------------------------------
+
+#Internal Modules------------------------------------------------------------------------------------
+from ravenframework.utils import InputData, InputTypes
+from ravenframework.utils import utils
+from .. import ValidationBase
+#Internal Modules End--------------------------------------------------------------------------------
+
+class Representativity(ValidationBase):
```

Comment:
There are two Representativity.py, I think you need to remove the one in framework/Models/PostProcessors/